src/lib/Components/Bidding/Screens/SelectMaxBid.tsx

```diff
@@ -39,7 +51,7 @@ export class SelectMaxBid extends React.Component<SelectMaxBidProps, SelectMaxBi
           selectedValue={this.state.selectedBidIndex}
         />
 
-        <Button text="NEXT" onPress={() => null} />
+        <Button text="NEXT" onPress={this.onPressNext} />
```

Comment:
careful with using this, I think it may need `this` re-binding, I know we have that in many other places